misk-hibernate/src/main/kotlin/misk/jdbc/DataSourceConfig.kt

```diff
@@ -99,7 +101,34 @@ data class DataSourceConfig(
         "jdbc:hsqldb:mem:${database!!};sql.syntax_mys=true"
       }
       DataSourceType.VITESS -> {
-        "jdbc:vitess://${config.host}:${config.port}/${config.database}"
+        var queryParams = ""
```

Comment:
I'd just do `queryParams = "?"` here. Should be legal to have a `?` with no params

Reply:
I'm gonna play it safe for now and leave as-is as I don't have a good way to test that `?` is safe (even though I agree w/ your assessment). Going to work on getting my test app hooked up to a vitess and then I'll try it out.